### DISCUSSION TITLE
Add a section onto plugin pages with the plugin name for plugin/install

### DISCUSSION THREAD

**Discussion by deanfields** [craftcms/cms#10158] [Ideas] [answered]
When attempting to run plugin/install from the command line I have to look in the plugin's folder to find what the name-space is for it, it's rarely the same as the composer install. it would be good if there were a section on the plugin pages like the 'Package name' section to make this easy to copy and paste.

**Comment by brandonkelly** [comment, marked as answer]
I’ll do you one better: as of the next release, the `plugin/install` action will no longer require that you pass a plugin handle immediately. When a handle isn’t supplied, it will output a list of uninstalled plugins that are present (names and handles), and have you choose one at runtime.

```sh
> php craft plugin/install
The following uninstalled plugins are present:

    Handle                 Name
    ---------------------  ---------------------
    anchors                Anchors
    apple-news             Apple News
    ckeditor               CKEditor
    contact-form           Contact Form
    contact-form-honeypot  Contact Form Honeypot
    commerce               Craft Commerce
    element-api            Element API
    feed-me                Feed Me
    fix-fks                Fix FKs
    gatsby-helper          Gatsby Helper
    google-cloud           Google Cloud Storage
    guest-entries          Guest Entries
    hexdec                 Hexdec
    mailgun                Mailgun
    mandrill               Mandrill
    postmark               Postmark
    rackspace              Rackspace Cloud Files
    simple-text            Simple Text
    store-hours            Store Hours

Choose a plugin handle to install: ckeditor
*** installing ckeditor
*** installed ckeditor successfully (time: 0.496s)
```

**Reply by deanfields** [reply]
Amazing!! Thank you

**Reply by brandonkelly** [reply]
3.7.23 is out now with that change. Went ahead and made a similar change to the `plugin/uninstall`, `plugin/enable`, and `plugin/disable` commands as well.

**Comment by siebird** [comment]
I just thought of an idea relating to this which brought me to this thread. 

It would be nice if the copy to clipboard button on the plugin page would prefix the `composer require` command with it. An added bonus would be if could run the install command too. So when you click the copy to clipboard button would copy the following command: 

`composer require craftcms/feed-me && php craft plugin/install feed-me`

**Reply by brandonkelly** [reply]
We’re actually working on an update to the Plugin Store which includes this :)

**Reply by siebird** [reply]
Awesome!
